src/test/java/net/openhft/chronicle/bytes/MappedFileTest.java

```diff
@@ -42,7 +42,11 @@
 
     @Test
     public void testWarmup() {
-        MappedFile.warmup();
+        try {
+            MappedFile.warmup();
+        } catch (Throwable t) {
+            fail(t.getMessage());
```

Comment:
Suggestion :  rewrite using JUnit5, ( something like assertNotThrows )  as this is just adding code clutter.